Assets/Scripts/DraggableBlock.cs

```diff
@@ -12,9 +12,18 @@ public class DraggableBlock : MonoBehaviour
     Block block;
     bool isDraggable = false;
 
+    Tile[,] tiles;
+    GameObject prefabTile;
+    Grid grid;
+    DraggableObject draggableObject;
+
+    //Copies the data of a Block into this DraggableBlock’s contained Block
     public void SetBlock(Block copiedBlock)
     {
-
+        GameObject tileObject = Instantiate(prefabTile);
```

Comment:
We don't need this line. TileUtil.CreateTileArray handles the Tile creation by itself.